src/status_im/init/core.cljs

```diff
@@ -137,22 +137,25 @@
 (fx/defn initialize-views
   [cofx]
   (let [{{:accounts/keys [accounts] :as db} :db} cofx]
-    (if (empty? accounts)
-      (navigation/navigate-to-cofx cofx :intro nil)
-      (let [account-with-notification
-            (when-not platform/desktop?
-              (notifications/lookup-contact-pubkey-from-hash
-               cofx
-               (first (keys (:push-notifications/stored db)))))
-            selection-fn
-            (if (not-empty account-with-notification)
-              #(filter (fn [account]
-                         (= account-with-notification
-                            (:public-key account)))
-                       %)
-              #(sort-by :last-sign-in > %))
-            {:keys [address photo-path name]} (first (selection-fn (vals accounts)))]
-        (accounts.login/open-login cofx address photo-path name)))))
+    (cond (empty? accounts)
+          (navigation/navigate-to-cofx cofx :intro nil)
+          (= 1 (count accounts))
```

Comment:
why 1 ? what if 2 ?

Reply:
Then we move to the `:else` clause. If i understand Figma design correctly, from now on we will show account selection in all cases when there is more than one account. But maybe this is wrong - @hesterbruikman can you please clarify? Good point however - i've removed the account selection logic for now.

Reply:
I would say desired behavior is as follows:

On launch...
- Show [Password/pin entry] for last used account
- If Save account was set for the last used account - skip unlock [Password/Pin entry]
- Account selection is shown when: user taps Back from [Password/Pin entry] OR when user selects [Log out] in [Profile > Settings]


cc @andmironov @maciej

Reply:
not sure If I can help anyhow

Reply:
Got it, thanks, fixing.

Reply:
@siphiuel so same as we had before, not changes needed here from UX pov

Reply:
> 
> 
> not sure If I can help anyhow

Sorry about that! I keep making that mistake:'( Have a great day

Reply:
>I would say desired behavior is as follows:

looks good to me!